core/math/geometry.h

```diff
@@ -31,6 +31,8 @@
 #ifndef GEOMETRY_H
 #define GEOMETRY_H
 
+#include "thirdparty/misc/clipper.hpp"
```

Comment:
Can we avoid having a thirdparty include in a core header? That would mean pulling clipper code everywhere `geometry.h` is included, I'd prefer that it stays only in the .cpp.

Reply:
Yeah... I added those for utility functions `_scale_up_polypaths`, `_scale_down_polypaths`, those could be taken away from `Geometry` declaration I think (was afraid to pollute global namespace for some reason).

Reply:
Used a `GodotClipperUtils` namespace but reduz wouldn't like that according to https://github.com/godotengine/godot/issues/30736#issue-470816808, so not sure. 😅